core/client.py

```diff
@@ -53,7 +53,12 @@
 _data_dict = None
 _file_ext = None
 
+
 class Client:
+    # It's unclear why, but sphinx refuses to generate method docs
```

Comment:
Do we want to keep these comments?

Reply:
I think any changes we do should be in the internal repo, not here. This PR is merely for syncing (not clear if this is the best way to do it, we should discuss).